### DISCUSSION TITLE
Render and edit vertexes of polyline on Google Map using Deckgl.

### DISCUSSION THREAD

**Discussion by ravikumar999** [visgl/deck.gl#5680] [General]
I am using Google Map to render polylines. I have a use case where users can render around 30k polylines also they can edit the vertex.  The browser gets freeze completely when I render those 30k polylines. I decided to use Deckgl with Google Maps. but I am not sure I could render those polylines and edit the vertex of those polylines on a deckgl layer.

I have created a codesanfbox https://codesandbox.io/s/dark-breeze-11fd6?file=/src/index.js

**Comment by padawannn** [comment]
You could use 
https://nebula.gl/geojson-editor/
https://nebula.gl/docs

It uses deck-gl. The problem is that it is not compatible with Google Map https://github.com/uber/nebula.gl/issues/286
